src/modules/apps/e2e/useInterval.tsx

```diff
@@ -0,0 +1,40 @@
+import { useRef, useEffect } from 'react'
```

Comment:
This file, `useInterval.tsx`, could be inside `report/` since it's used only there